public/admin/config.yml

```diff
@@ -99,6 +85,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
+          name: "Category",
+          widget: "select",
+          options:
+            [
+              "admin-and-info",
+              "media-publicity-events",
+              "working-with-people",
+              "working-with-the-authorities",
+            ],
+          default: "admin-and-info",
```

Comment:
Probably don't want a default. Would be too easy to add to wrong category by accident.

Reply:
done